package.json

```diff
@@ -349,6 +349,7 @@
     "@types/parse-torrent": "^5.8.3",
     "@types/react-router-dom": "^5.1.9",
     "@types/webtorrent": "^0.109.0",
+    "@solana/web3.js": "^0.30.8",
```

Comment:
can you pin the exact version here? given that this dependency gets used in webpages, i think we want to audit any updates to it

Reply:
done, pinned the latest release 1.36.0